components/camera/videosource/webcam.go

```diff
@@ -594,13 +596,52 @@ func (c *monitoredWebcam) NextPointCloud(ctx context.Context) (pointcloud.PointC
 	return c.exposedProjector.NextPointCloud(ctx)
 }
 
+func (c *monitoredWebcam) DriverInfo() (driver.Info, error) {
+	c.mu.RLock()
+	defer c.mu.RUnlock()
+	if c.underlyingSource == nil {
+		return driver.Info{}, errors.New("no configured camera")
+	}
+	d, err := gostream.DriverFromMediaSource[image.Image, prop.Video](c.underlyingSource)
+	if err != nil {
+		return driver.Info{}, errors.Wrap(err, "cannot get driver from media source")
+	}
+	return d.Info(), nil
+}
+
+//go:embed data/intrinsics.json
+var intrinsics []byte
+
 func (c *monitoredWebcam) Properties(ctx context.Context) (camera.Properties, error) {
 	c.mu.RLock()
 	defer c.mu.RUnlock()
 	if err := c.ensureActive(); err != nil {
 		return camera.Properties{}, err
 	}
-	return c.exposedProjector.Properties(ctx)
+
+	props, err := c.exposedProjector.Properties(ctx)
+	if err != nil {
+		return camera.Properties{}, err
+	}
+	if props.IntrinsicParams == nil {
+		dInfo, err := c.DriverInfo()
+		if err != nil {
+			return camera.Properties{}, errors.Wrap(err, "can't find driver info for camera")
```

Comment:
should this return an error? If for some reason the driver info cannot be found, does that prevent the program from continuing to run? Potentially make this a log warning

Reply:
I agree. Done